package.json

```diff
@@ -43,7 +43,7 @@
     "commander": "^2.19.0",
     "doctrine": "^3.0.0",
```

Comment:
We should update `@babel/core` to `^7.4.4` so we ensure that typescript is fully supported.

Also noticed we have it in dependencies and dev dependencies. Having it only in dependencies is probably fine.